src/consumerDrivenContractTest/java/uk/gov/hmcts/reform/bulkscanprocessor/contract/tests/BulkScanProcesserConsumerDetailsTest.java

```diff
@@ -0,0 +1,100 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.contract.tests;
+
+import au.com.dius.pact.consumer.Pact;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactConsumerTestExt;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.model.RequestResponsePact;
+import com.google.common.collect.ImmutableList;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.util.ResourceUtils;
+import uk.gov.hmcts.reform.bulkscanprocessor.ocrvalidation.client.OcrValidationClient;
+import uk.gov.hmcts.reform.bulkscanprocessor.ocrvalidation.client.model.req.FormData;
+import uk.gov.hmcts.reform.bulkscanprocessor.ocrvalidation.client.model.req.OcrDataField;
+import uk.gov.hmcts.reform.bulkscanprocessor.ocrvalidation.client.model.res.Status;
+import uk.gov.hmcts.reform.bulkscanprocessor.ocrvalidation.client.model.res.ValidationResponse;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+
+import static io.pactfoundation.consumer.dsl.LambdaDsl.newJsonBody;
+import static java.util.UUID.randomUUID;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+
+@ExtendWith(PactConsumerTestExt.class)
+@PactTestFor(providerName = "ocr_validation_service", port = "8889")
+@SpringBootTest(properties = {
+    "ocr_validation_service: localhost:8889",
+    "ocr_validation_service.api.url: localhost:8889"})
+@ContextConfiguration(classes = ConsumerApplication.class)
+public class BulkScanProcesserConsumerDetailsTest {
```

Comment:
typo in 'processor'

Reply:
would be good if this somehow indicated that it's testing the validation endpoint